modules/libuuid/2.39.3-20240204/patches/add_build_file.patch

```diff
@@ -0,0 +1,63 @@
+diff --git BUILD.bazel BUILD.bazel
+new file mode 100644
+index 000000000..c960cbd65
+--- /dev/null
++++ BUILD.bazel
+@@ -0,0 +1,57 @@
++cc_library(
++    name = "common",
++    srcs = [
++        "lib/md5.c",
++        "lib/randutils.c",
++        "lib/sha1.c",
++        "lib/strutils.c",
++    ],
++    hdrs = [
++        "include/all-io.h",
++        "include/bitops.h",
++        "include/c.h",
++        "include/md5.h",
++        "include/nls.h",
++        "include/pathnames.h",
++        "include/randutils.h",
++        "include/sha1.h",
++        "include/strutils.h",
++    ],
++    defines = ["HAVE_NANOSLEEP"],
++    strip_include_prefix = "include",
++    visibility = ["//visibility:public"],
++)
++
++cc_library(
++    name = "uuid",
++    srcs = [
++        "libuuid/src/uuidP.h",
++        "libuuid/src/clear.c",
++        "libuuid/src/compare.c",
++        "libuuid/src/copy.c",
++        "libuuid/src/gen_uuid.c",
++        "libuuid/src/isnull.c",
++        "libuuid/src/pack.c",
++        "libuuid/src/parse.c",
++        "libuuid/src/predefined.c",
++        "libuuid/src/unpack.c",
++        "libuuid/src/unparse.c",
++        "libuuid/src/uuidd.h",
++        "libuuid/src/uuid_time.c",
++    ],
++    deps = [
++        ":common",
++    ],
++    hdrs = [
++        "libuuid/src/uuid.h",
++    ],
```

Comment:
For "uuid" cc_library, please add after this line to suppress the compiler warnings:
```
    copts = [
        "-Wno-implicit-function-declaration",
        "-Wno-unused-parameter",
    ],
```